app/livechat/client/views/app/livechatReadOnly.js

```diff
@@ -20,9 +19,12 @@ Template.livechatReadOnly.helpers({
 		return room && room.open === true;
 	},
 
-	guestPool() {
-		return settings.get('Livechat_Routing_Method') === 'Guest_Pool';
+	showPreview() {
+		const config = Template.instance().routingConfig.get();
+		const { previewRoom } = config;
+		return previewRoom;
```

Comment:
Why not just `return config.previewRoom`? And is it possible to config not be an object?

Reply:
The `config` will always be an object.
Done.